examples/build.gradle

```diff
@@ -14,6 +30,22 @@ dependencies {
     implementation project(path: ':grpc-utils')
 }
 
+protobuf {
+    protoc {
+        artifact = "com.google.protobuf:protoc:${protocVersion}"
+    }
+
+    plugins {
+        grpc { artifact = "io.grpc:protoc-gen-grpc-java:${grpcVersion}" }
+    }
+
+    generateProtoTasks {
+        all()*.plugins {
```

Comment:
do you know what `all()` here does?

Reply:
I re-used it from https://github.com/grpc/grpc-java/blob/4a137d6ef0415b4e4451ee01718cddf629b636e7/examples/build.gradle#L50. Can I let change it in another future commit behind this one?

Reply:
okay:)